src/components/AssetDialog.tsx

```diff
@@ -0,0 +1,43 @@
+import AssetListItem from 'components/AssetListItem';
+import ListItem from 'components/ListItem';
+import SkeletonListItem from 'components/SkeletonListItem';
+import type { Brand } from 'store/app';
+
+const AssetDialog = ({
+  brands,
+  handleBrandSelected,
+}: {
+  brands: Array<Brand>;
+  handleBrandSelected: (brand: Brand) => void;
+}) => {
+  if (!(brands && brands.length))
+    return (
+      <div className="flex flex-col gap-4 p-5 overflow-auto ">
+        {[...Array(4).keys()].map(i => (
```

Comment:
this skeleton section is always the same so no need to execute it within this function. please move it up to a `const skeleton: JSX.Element`.